src/screens/AuthScreen/AuthScreen.tsx

```diff
@@ -19,20 +19,25 @@ import { AuthContext } from '../../context/AuthContext';
 import { getUserData, goalsAuth, requestCameraPermission } from './Util';
 import { storeData } from '../../utils/dataStore';
 import AuthApis from '../../constants/apiConstant/AuthApi';
-// import { AuthApisStaging } from '../../constants/apiConstant/AuthApi';
-import { CameraScreen } from 'react-native-camera-kit';
+import { Camera } from 'react-native-camera-kit';
```

Comment:
Why is this changed to the camera?

Reply:
You can check here - 
https://github.com/teslamotors/react-native-camera-kit?tab=readme-ov-file#barcode--qr-code-scanning

Please check that CameraScreen was removed - https://github.com/teslamotors/react-native-camera-kit/commit/5a7df3ee804c189570099426b43cc9676ee6b0b5